docs/user-guide/nextcloud-integration/README.md

```diff
@@ -174,6 +174,18 @@ There are two additional features related to the integration that you can enable
 
 ![Nextcloud search for work packages](nc_global_search.png)
 
+## Work package link preview in Nextcloud
+
+Starting with OpenProject 13.0 a work package link preview will be shown if you use Nextcloud Talk or Text apps. You can [copy a work package link](../work-packages/copy-move-delete/#copy-link-to-clipboard) and paste it into a Talk or Text app. Alternatively you can use **/** to activate the **smart picker** and find the work package by searching.
```

Comment:
@MayaBerd Actually the link preview is a feature purely on the Nextcloud side. So the relevant version is the one of the Nextcloud app (2.4) and not OpenProject

Reply:
making the change here: https://github.com/opf/openproject/pull/13528